.github/workflows/build_conda_windows.yml

```diff
@@ -169,6 +169,14 @@ jobs:
             --no-test  --python "${PYTHON_VERSION}" \
             --output-folder distr/ \
             "${CONDA_PACKAGE_DIRECTORY}"
+
+          # This is to remove the vs20xx conda package that sneaks into distr
+          for pkg in distr/**/*.tar.bz2; do
+            if [[ "${pkg}" == *vs20* ]]; then
```

Comment:
Should we instead remove everything that does not match:
``
$PACKAGE_NAME.*.tar.bz2 
`` ?

Reply:
Sounds good, I didn't know that we have `$PACKAGE_NAME` ready to use in the build job.